cart/src/App.js

```diff
@@ -0,0 +1,79 @@
+import React, { useEffect, useState } from "react";
+import Cart from "./components/Cart";
+import Home from "./components/Home";
+import NavBar from "./components/Navbar";
+
+const App = () => {
+  const [showScreen, setShowScreen] = useState(true);
```

Comment:
this state variable name is not clear, I don't understand clearly what it's purpose is unless I dig into the code. Pls add a clear name & update the prop name as well where this is used